test/functional/test_framework/util.py

```diff
@@ -180,6 +180,18 @@ def assert_array_result(object_array, to_match, expected, should_not_find=False)
     if num_matched > 0 and should_not_find:
         raise AssertionError("Objects were found %s" % (str(to_match)))
 
+def assert_scale(number, expected_scale=8):
+    """Assert number has expected scale, e.g. fractional digits; number of
+    digits after the decimal. The default of 8 corresponds to a Bitcoin amount."""
+    number = str(number)
+    mantissa = number.split('.')[-1]
+    if mantissa[:3] == '0E-':
+        assert_equal(mantissa, '0E-{}'.format(expected_scale))  # exponent notation
```

Comment:
I'm not a python expert, but my python 3.6.9 prints numbers in scientific notation using "e" rather "E".
Btw, in what cases we could expect values with scientific notation in RPC responses?

Reply:
In every run of `interface_bitcoin_cli.py` the tests see zero values like `'paytxfee': Decimal('0E-8'),` in the -getinfo response. I see them as well if I add logging like `self.log.info(cli_get_info)` in the test.

Reply:
Example:
```
2020-06-06T16:11:48.077000Z TestFramework (INFO): Test -getinfo returns expected network and blockchain info
{
 'balance': Decimal('50.00000000'),
 'blocks': 101,
 'chain': 'regtest',
 'connections': 0,
 'difficulty': Decimal('4.656542373906925E-10'),
 'headers': 101,
 'keypoolsize': 1,
 'paytxfee': Decimal('0E-8'),
 'proxy': '',
 'relayfee': Decimal('0.00001000'),
 'timeoffset': 0,
 'unlocked_until': 0,
 'verificationprogress': 1,
 'version': 209900
}
```

Reply:
Added `upper()` just in case.